sdk/core/Azure.Core/tests/ClientDiagnosticsTests.Net50.cs

```diff
@@ -165,24 +162,6 @@ public void NestedClientActivitiesSuppressed(int kind, bool expectSuppression)
             Assert.AreEqual("ClientName.ActivityName", Activity.Current.OperationName);
         }
 
-        [Test]
-        [NonParallelizable]
-        public void NestedClientActivitiesSuppressedDefaultDiagnosticScopeFactory()
```

Comment:
Why remove these tests?

Reply:
It was testing default value of suppression flag in  `DiagnosticScopeFactory` which I removed. There is no default value anymore and we have tests for explicit values.